clang/test/CodeGenCXX/builtin-calling-conv.cpp

```diff
@@ -13,7 +13,7 @@ using size_t = unsigned long;
 #endif // SPIR
 } // namespace std
 
-float __builtin_atan2f(float, float);
+float __builtin_erff(float);
```

Comment:
Just want to put a note here we are doing this because atan2 would no longer test the calling convention intended.

Reply:
Right, I hope I captured that in the PR description above.  If that's not clear enough let me know.